courses/views_test.py

```diff
@@ -486,6 +486,25 @@ def test_201_on_happy_case_with_modules(self):
                 content_type="application/json"
             )
         assert result.status_code == 201, result.content.decode('utf-8')
+        # check that the payload has been sent to edx as json
+        # this means that the requests library will encode it properly
+        # and add the `application/json` content type header
+        call_to_edx = mock_req.post.call_args
+        call_content = call_to_edx[1]
+        assert 'json' in call_content
+        paylod_to_edx = {
+            'course_modules': [course_module.locator_id for course_module in course_modules],
+            'coach_email': payload['user_email'],
+            'master_course_id': course.course_id,
+            'display_name': payload['display_name'],
+            'max_students_allowed': payload['total_seats']
+        }
+        for key, value in paylod_to_edx.iteritems():
```

Comment:
I think you need to use `items` here instead of `iteritems` since Python 3 doesn't have `iteritems`